src/workingtitle-vcockpits-instruments-navsystems/html_ui/Pages/VCockpit/Instruments/NavSystems/AS1000/MFD/AS1000_MFD.html

```diff
@@ -929,6 +929,8 @@
 <link rel="stylesheet" href="AS1000_MFD_WT.css" />
 
 <script type="text/html" import-script="/JS/SimPlane.js"></script>
+<script type="text/html" import-script="/JS/dataStorage.js"></script>
```

Comment:
While looking at some other stuff, I have noticed that we may be able use JS imports like this:

```
Include.addScript("/JS/Services/Aircraft.js");
```

That would allow us to pull dataStorage.js in CommonPFD_MFD.js and avoid changing every *.html files we want to use it and also remove the try/catch blocks in PersistVar.

Reply:
Interesting idea and I think I like it.  Any thoughts on that, @dga711 , @mattnischan ?

Reply:
opened an issue for us to come back to that, thanks, @tavip!

Reply:
> Interesting idea and I think I like it. Any thoughts on that, @dga711 , @mattnischan ?

Yeah. Actually. I was thinking about also putting this as a comment to include it that way. But then I was like "eh, you have changed the html already anyway" ^^
But yeah, @tavip is right

Reply:
The only downside I can see is that it might not allow us to cache bust as easily.

Reply:
@mattnischan luckily addscript runs through the same hoops as the includes in the script-tags.